jbpm-designer-client/src/main/resources/org/jbpm/designer/public/stencilsets/bpmn2.0jbpm/stencildata/bpmn2.0jbpm.orig

```diff
@@ -3812,8 +3812,8 @@
 					"optional":true
 					 $if(workitemDefs.(k).customEditor)$
 					,
-					"hidden":true,
-					"visible":false
+					"hidden":false,
+					"visible":true
```

Comment:
This means that even when a service task has  customEditor set to true, the 'assignments' property is displayed, and the task has the button to bring up the DataIOEditor. Values for custom properties can be edited either in the custom property field or in the DataIOEditor, and the two are kept in sync. For example, if you update a custom property in the DataIOEditor, when the dialog is closed, the property field is updated with the new value.